src/redux/stores/balances/balancesHelpers.ts

```diff
@@ -73,9 +74,11 @@ export const getOwnedTokenIdsOfWallet = async (
 
     /* get unique values */
     const uniqueTokenIds = foundTokenIds
-      .map(t => t.toNumber())
+      .sort((a, b) => a.sub(b).toNumber())
```

Comment:
optionnal: could be extracted as a method to factorize line 50

Reply:
good point, added getUniqueTokenIds function